packages/cover-traffic/src/unreleasedTokens.spec.ts

```diff
@@ -0,0 +1,40 @@
+import assert from 'assert'
+import BigNumber from 'bignumber.js'
+import PeerId from 'peer-id'
+import { utils } from 'ethers'
+import UNRELEASED_TOKENS from './unreleasedTokens.json'
+
+describe('test unreleasedTokens.json', function () {
+  const unreleasedTokens: {
+    ethAddress: string
+    hoprId: string
+    tokens: string
+  }[] = UNRELEASED_TOKENS
+
+  it('should validate entries', function () {
+    for (const { ethAddress, hoprId, tokens } of unreleasedTokens) {
+      assert(utils.isAddress(ethAddress), `invalid ethAddress: ${ethAddress}`)
+
+      let validHoprId = false
+      let errMsg = ''
+      try {
+        PeerId.createFromB58String(hoprId)
+        validHoprId = true
+      } catch (err) {
+        errMsg = err
+      }
+      assert(validHoprId, `invalid hoprId: ${hoprId}, ${errMsg}`)
```

Comment:
This does not check if the HoprId matches to the given Ethereum address

Reply:
it doesnt need to

Reply:
I assume the overall idea of this test case is to filter typos such as unintended copy-paste errors, so it should - at least imo - not only test if the hoprId is valid but also if ethAddress and hoprId fit to each other

Reply:
they dont need to fit each other, please read issue description